spec/merge_spec.rb

```diff
@@ -199,5 +199,21 @@ class MyRandomclass
     end      
      
     
+    it "should not merge attributes that are included in the ignore_attributes array" do
+      @A.array_hashes = [{test1: "test1"}]
+      @B.array_hashes = [{test2: "test2"}]
+      @A.merge!(@B, {}, ['array_hashes'])
+
+      @A.array_hashes.size.should == 1
+    end
+
+    it "should  merge attributes that are not included in the ignore_attributes array" do
+      @A.array_simple_types = ["test1"]
+      @B.array_simple_types = ["test2"]
```

Comment:
No special reason. Just wanted to show that the feature works with different document fields.